### PR TITLE
[risk=low][RW-9518] Add some AppsPanel unit tests

### PR DESCRIPTION
**Pull request by jmthibault79**
Plus a few minor code updates

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: ui/src/app/components/apps-panel.spec.tsx [hunk 1]
```diff
@@ -1,64 +1,82 @@
...
     registerApiClient(RuntimeApi, runtimeStub);
     runtimeStore.set({
-      workspaceNamespace: workspaceDataStub.namespace,
+      workspaceNamespace: workspaceStub.namespace,
       runtime: runtimeStub.runtime,
       runtimeLoaded: true,
     });
   });
 
-  it('should render', async () => {
```
Comment (jmthibault79) [inline ui/src/app/components/apps-panel.spec.tsx:56]: these kinds of test cases are pretty useless, and often misleading

File: ui/src/app/components/apps-panel.spec.tsx [hunk 2]
```diff
@@ -120,4 +134,134 @@ describe('AppsPanel', () => {
...
+      appsStub.listAppsResponse = [{ status, appType: AppType.CROMWELL }];
+
+      const wrapper = await component();
+      expect(wrapper.exists()).toBeTruthy();
+      await waitOneTickAndUpdate(wrapper);
+
+      expect(findActiveApps(wrapper).exists()).toBe(activeExpected);
+      expect(findAvailableApps(wrapper, activeExpected).exists()).toBe(
+        availableExpected
+      );
```
Comment (Peter-Lavigne) [inline ui/src/app/components/apps-panel.spec.tsx:167]: Could `availableExpected` be removed as a parameter? It's always `true` for these tests.

**Review by Peter-Lavigne — APPROVED** [review]
LGTM! Thanks for doing this.

File: ui/src/app/components/apps-panel.spec.tsx (hunk 2 above)
Comment (jmthibault79) [reply ui/src/app/components/apps-panel.spec.tsx:167]: good point!  I'll merge this as-is but I'll think about that on the branch I'm updating

Comment (jmthibault79) [reply ui/src/app/components/apps-panel.spec.tsx:167]: yeah this change looks good so it will be in a future PR

**Pull request merged**